test/powershell/Language/Scripting/DeserializedTypeConversion.Tests.ps1

```diff
@@ -94,7 +94,7 @@ Describe "Tests conversion of deserialized types to original type using object p
         }
 
         It 'Type casts should succeed.' {
-            { $tc1 = [test1]$dst1 }| Should not Throw
+            { $tc1 = [test1]$dst1 }| Should -Not -Throw
```

Comment:
The `Should -Not -Throw` pattern is not needed any more since Pester v4.2 (see release notes [here](https://github.com/pester/Pester/releases/tag/4.2.0)):
> Should -Not -Throw will fail if any exception is thrown. The filters applied on Should -Not -Throw do not apply, because when an exception is thrown, it automatically means that the test should not pass.